src/huak/env/mod.rs

```diff
@@ -1,3 +1,4 @@
-mod system; // TODO: Is Venv a trait of Env?
+/// This currently only exposes find_python_binary_paths
```

Comment:
Is this comment necessary?

Reply:
I initially thought that was the only thing I had time for so I left a comment there for later.